editor-components/Image_gallery/image_gallery_7/image-gallery7.tsx

```diff
@@ -0,0 +1,490 @@
+                    {
+                        type: "array",
+                        key: "card",
+                        displayer: "Card 3",
+                        value: [
+                            { 
+                                type: "image",
+                                key: "image",
+                                displayer: "Image 3",
+                                value: "https://storage.googleapis.com/download/storage/v1/b/hq-composer-0b0f0/o/669e003a2f8a5b002ce6012d?alt=media", 
+                            },
+                            {
+                                type: "string",
+                                key: "title",
+                                displayer: "title",
+                                value: "Ron Mcclenny",
+                            },
+                            {
+                                type: "string",
+                                key: "subtitle",
+                                displayer: "subtitle",
+                                value: "Portraits, Summer"
+                            },
+                            {
+                                type:"color",
+                                key:"backgroundColor",
+                                displayer:"Background Color",
+                                value:"#f1f1e8"
+                            }
+                        ]
+                    },
+                    {
+                        type: "array",
+                        key: "card",
+                        displayer: "Card 4",
+                        value: [
+                            { 
+                                type: "image",
+                                key: "image",
+                                displayer: "Image 4",
+                                value: "https://storage.googleapis.com/download/storage/v1/b/hq-composer-0b0f0/o/669e005b2f8a5b002ce60139?alt=media", 
+                            },
+                            {
+                                type: "string",
+                                key: "title",
+                                displayer: "title",
+                                value: "Curitiba Brasil",
+                            },
+                            {
+                                type: "string",
+                                key: "subtitle",
+                                displayer: "subtitle",
+                                value: "Architecture, Interior"
+                            },
+                            {
+                                type:"color",
+                                key:"backgroundColor",
+                                displayer:"Background Color",
+                                value:"#f1f1e8"
+                            }
+                        ]
+                    }, 
+                    {
+                        type: "array",
+                        key: "card",
+                        displayer: "Card 5",
```

Comment:
use singular and non-contain number names.
```suggestion
                        displayer: "Card",
```